CRM/Core/Session.php

```diff
@@ -152,10 +152,21 @@ public function initialize($isRead = FALSE) {
   /**
    * Resets the session store.
    *
-   * @param int $all
+   * @param int|string $mode
+   *   1: Default mode. Deletes the `CiviCRM` data from $_SESSION.
+   *   2: More invasive version of that. (somehow)
```

Comment:
ah.. my attempt at this definitely floundered on trying to work out what these were all about...